src/steps/Microsoft.DotNet.UpgradeAssistant.Steps.Packages/AnalyzePackageStatus.cs

```diff
@@ -84,18 +84,21 @@ private static IReadOnlyCollection<string> ExtractAnalysisResult(IDependencyAnal
             }
             else
             {
-                GetResults("References to Delete", analysisState.References.Deletions);
-                GetResults("References to Add", analysisState.References.Additions);
-                GetResults("Packages to Delete", analysisState.Packages.Deletions);
-                GetResults("Packages to Add", analysisState.Packages.Additions);
-                GetResults("Framework References to Delete", analysisState.FrameworkReferences.Deletions);
-                GetResults("Framework References to Add", analysisState.FrameworkReferences.Additions);
+                GetResults("Reference to ", " needs to be deleted ", analysisState.References.Deletions);
```

Comment:
I think it would be nice to have a separate rule for reference additions/deletions, package additions/deletions, etc. Not a blocker for now, but a thought.

Reply:
I'll keep that in mind for the next iteration.